vrk-kpa/suomidigi
# SUOPA-724 Blog posts

To test: 
- `make fresh`
- `make shell`
- Run Blog post conversion command via drush `drush convert-blogs && drush cr`
- Go to https://suomidigi.docker.sh/admin/content?title=&type=blog_post&article_type=All&status=All&langcode=All and check that all blog posts are now in their own content type. Click through the blog posts and check that you can edit them and that they look like before.
